src/atoms/label/Label.html

```diff
@@ -1,5 +1,5 @@
 <label
-  class="a-label"
+  :class="getClass('label')"
```

Comment:
when the element has only one attribute, then we don't split lines in general. can we stick to this convention?